samples/maps/demo/flowmap-ship-route/demo.js

```diff
@@ -0,0 +1,158 @@
+(async () => {
+    const topology = await fetch(
+        'https://code.highcharts.com/mapdata/custom/world.topo.json'
+    ).then(response => response.json());
+
+    Highcharts.mapChart('container', {
+        chart: {
+            map: topology
+        },
+
+        title: {
+            text: 'Example route of the ship from USA to China'
+        },
+
+        subtitle: {
+            text: `The route through the Suez Canal with a number of travel days
+            between cities`
+        },
+
+        mapNavigation: {
+            enabled: true
+        },
+
+        mapView: {
+            center: [10, 20],
+            zoom: 2.2
```

Comment:
On mobile the flows are cropped. We really should have `mapView.bounds` option...

Reply:
Implemented here: https://github.com/highcharts/highcharts/pull/18123